desktop/webpack.config.development.js

```diff
@@ -22,6 +25,31 @@ config.output.publicPath = 'http://localhost:4000/dist/'
 // Uncomment below to figure out packaging bugs
 // config.bail = true
 
+config.plugins.push(new UnusedFilesWebpackPlugin({
+  pattern: '../shared/**/*.js',
```

Comment:
have to do this else the paths won't match up due to symlink issues. its solvable by forking the plugin but it also makes it slower to resolve them